ci-operator/config/openshift-knative/eventing-kafka-broker/openshift-knative-eventing-kafka-broker-release-v1.3__49.yaml

```diff
@@ -275,6 +275,8 @@ tests:
         name: knative-eventing-kafka-broker-dispatcher
       - env: KNATIVE_EVENTING_KAFKA_BROKER_KAFKA_CONTROLLER
         name: knative-eventing-kafka-broker-kafka-controller
+      - env: KNATIVE_EVENTING_KAFKA_BROKER_POST_INSTALL
+        name: knative-eventing-kafka-broker-post-install
```

Comment:
should we call it more like `knative-eventing-kafka-controller-post-install` ?

No need to do it now - but general fruit for thought